container/container_windows.go

```diff
@@ -8,6 +8,11 @@ import (
 	"path/filepath"
 
 	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/pkg/system"
+)
+
+const (
+	containerInternalConfigsDirPath = `C:\ProgramData\Docker\internal\configs`
```

Comment:
We made a mistake choosing `programdata` over `program files` in the daemon due to the ACLing. I think this should be under `program files` so we don't have to fix later, and have the right ACLs when running a container as a regular non-admin user.